en/docs/manage/cloud/rag-ingestion/retrieval.md

```
@@ -15,7 +15,7 @@ Retrieval-augmented generation (RAG) retrieval searches a vector store for the m
 
 To retrieve chunks that have already been ingested, open your organization from the **Organization** dropdown in the console header. In the left navigation menu, click **RAG**, then select **Retrieval**.
 
-### Step 1: Initialize the vector store
+### Step 1: initialize the vector store
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)